src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -201,6 +223,15 @@ internal abstract partial class BaseArgument : Operation, IArgument
         /// Conversion applied to the argument value after the invocation. Applicable only to VB Reference arguments.
         /// </summary>
         public abstract IOperation OutConversion { get; }
+        public override IEnumerable<IOperation> Children
+        {
+            get
+            {
+                yield return Value;
```

Comment:
Are we exercising the new `Children` implementations from existing tests? 
It seems like the order of returned children is significant and so should be tested.